tapir/shifts/templates/shifts/shift_form.html

```diff
@@ -15,18 +15,28 @@
             <h5 class="card-header d-flex justify-content-between align-items-center">
                 {{ card_title }}
             </h5>
-            <div class="card-body">
-                <form role="form" method="post">
-                    {% csrf_token %}
-                    {% bootstrap_form form %}
-                    <div class="d-flex justify-content-end">
-                        <button type="submit" class="btn tapir-btn btn-primary" id="shift_save_button">
-                            <span class="material-icons">save</span>
-                            {% translate 'Save' %}
-                        </button>
-                    </div>
-                </form>
-            </div>
+            <ul class="list-group list-group-flush">
+                {% if not form.instance.pk %}
+                    <li class="list-group-item">
+                        {% blocktranslate %}
+                            <p>Be careful when creating a shift, as you won't be able to delete it afterwards.</p>
```

Comment:
Will it be possible to delete shifts in the future or is it somehow prohibited by the user rights? If first is the case, maybe we could give some hint ("not yet possible to delete")

Reply:
If we have more shift management pages (for example adding shift templates, generating shifts...), I would create a separate page for them to avoid having too many links in the sidebar. But they would still be separate links, this one is only for creating a shift.